server/models/ipfspins.ts

```diff
@@ -0,0 +1,42 @@
+import * as Sequelize from 'sequelize';
+import { Model, DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+export type IpfsPinsAttributes = {
+  id: number;
+  address_id: number;
+  ipfs_hash: string;
+  created_at: Date;
+  updated_at: Date;
+};
+
+export type IpfsPinsInstance = ModelInstance<IpfsPinsAttributes>;
+
+export type IpfsPinsModelStatic = ModelStatic<IpfsPinsInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): IpfsPinsModelStatic => {
+  const IpfsPins = <IpfsPinsModelStatic>sequelize.define(
+    'IpfsPins',
+    {
+      id: { type: dataTypes.INTEGER, primaryKey: true },
+      address_id: { type: dataTypes.INTEGER, allowNull: false },
```

Comment:
Is `address_id` the `id` field from an `Address`? If so, you need to declare an association between the tables, like in `DiscussionDrafts`. Update this in the migration too.

Reply:
@zakhap, would love some clarification on this

Reply:
Done

Reply:
It is in the migration as well: https://github.com/hicommonwealth/commonwealth/pull/1730/commits/3672ef617308b155f4a076003d1fa808c2fda77d